setup.cfg

```diff
@@ -25,7 +25,7 @@ install_requires =
     Flask-Mail>=0.9.1
     Flask-Principal>=0.4.0
     Flask-WTF>=1.1.0
-    Flask>=2.0
+    Flask>=1.1.4,<2.3.0
```

Comment:
what was the reason of relaxing the lower pinning from 2.0 to 1.1.4?

Reply:
I discussed this with @slint since CI was failing and he suggested to make it the same as invenio-base 

https://github.com/inveniosoftware/invenio-base/blob/92b07ccddfa648e3ca2835ba958c717bb16400db/setup.cfg#L31